ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -127,6 +131,55 @@ + (NSString *)findMainBundleInFolder:(NSString *)folderPath
     return nil;
 }
 
++ (NSString *)getDefaultAssetsFolderName
+{
+    return DefaultAssetsFolderName;
+}
+
++ (NSString *)getDefaultJsBundleName
+{
+    return DefaultJsBundleName;
+}
+
++ (NSString *)getHashForBinaryContents:(NSURL *)binaryBundleUrl
```

Comment:
There seems to be a decent amount of duplication between this and verifyHashForDiffUodate. Is it possible to refactor them to share some of this logic?